README.md

```diff
@@ -12,6 +12,22 @@ Ultimately, it aims to support web extensions in a standard, portable,
 cross-platform way. Initially, it will provide a streamlined experience for developing
 [Firefox web extensions](https://developer.mozilla.org/en-US/Add-ons/WebExtensions).
 
+Table of Contents (ToC)
```

Comment:
This is a great idea but it will be hard to maintain so I'd rather not include it unless you can fully automate it.

I allowed a TOC in [the contributing doc](https://raw.githubusercontent.com/mozilla/web-ext/master/CONTRIBUTING.md) and we have instructions on how to update it. I still don't really like that approach though because it's easy to forget to run the generator script.